lib/default.nix

```diff
@@ -1,4 +1,5 @@
 # Args probably only needs pkgs and lib
+# TODO: build `lib` without needing `pkgs`
```

Comment:
Were you planning on addressing this in this PR or a follow up?

Reply:
That'll need some more thought. The "whole" lib includes the mkNixvimTest functions that all also expect `pkgs`, and this is all publicly documented and in the standalone flake template...

I don't have a firm idea, but my general thought is that we'll come up with new functions that don't require `pkgs`, then transition away from the old API after 24.11?